Jenkinsfile_CNP

```diff
@@ -60,9 +60,15 @@ withPipeline("nodejs", product, component) {
     yarnBuilder.yarn('setup')
     yarnBuilder.yarn('generate-excel-all')
     yarnBuilder.yarn('generate-bulk-excel-all')
+    env.CCD_ADMIN_URL="https://admin-web-div-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
   }
 
   after('functionalTest:preview') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test/functional/output/*'
   }
+
+  before('functionalTest:aat') {
+    steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test/functional/output/*'
```

Comment:
this should be archiving the XLSX, not the test output, also add this to the before of preview if your want the configs to be archived